To-do list/Models/StatusButton.swift

```diff
@@ -0,0 +1,38 @@
+//
+//  StatusButton.swift
+//  To-do list
+//
+//  Created by Kyrylo Tokar on 2022-09-16.
+//
+
+import UIKit
+
+class StatusButton {
+    
+    static func updateStatus(_ sender: UIButton, _ status: Task.Status) {
+        sender.setTitle(status.rawValue, for: .normal)
+        
+        switch status {
+        case .inProgress:
+            sender.backgroundColor = .systemOrange
+        case .done:
+            sender.backgroundColor = .systemRed
+        case .todo:
+            sender.backgroundColor = .systemGreen            
```

Comment:
You should not pass UI elements in the Models

Reply:
moved